src/Main.js

```diff
@@ -0,0 +1,48 @@
+import React, { Component } from 'react'
+import Menu from './components/Menu/Menu';
+import { getAllGoods } from './request';
+
+export class Main extends Component {
+      constructor(props) {
+    super(props);
+    this.state = {
+      currentProducts: {},
+      category: 'Закуски'
+    };
+    this.chooseCategory = this.chooseCategory.bind(this);
+  }
+  async componentDidMount() {
+    const { category } = this.state;
+    const { data } = await getAllGoods();
+    const currentProducts = data.categories.find((el) => el.name === category);
+   
+    this.setState(() => ({
+      currentProducts,
+      data
+    }));
+  }
+  chooseCategory(category) {
+    this.setState({
+      category,
+      currentProducts: this.state.data.categories.find((el) => el.name === category),
+    });
+  }
+  render() {
+   const { currentProducts} = this.state;
+    return (
+      Object.keys(currentProducts).length
```

Comment:
Remove "not yet" string. Instead create loader component. Might be your ox or default loader.

Reply:
Done